docs/src/tutorial.md

```diff
@@ -0,0 +1,350 @@
+This tutorial will walk you through its main functionality.
+That is, given a `DynamicalSystem` instance, find all its attractors and their basins
+of attraction. Then,
+continue these attractors, and their stability properties, across a parameter value.
+It also offers various functions that compute nonlocal stability properties for an
+attractor, any of which can be used in the continuation to quantify stability.
+
+Besides this main functionality, there are plenty of other stuff,
+like for example [`edgestate`](@ref) or [`basins_fractal_dimension`](@ref),
+but we won't cover anything else in this introductory tutorial.
+See the [examples](@ref examples) page instead.
+
+## Input: a `DynamicalSystem`
+
+The key input for most functionality of Attractors.jl is an instance of
+a `DynamicalSystem`. If you don't know how to make
+a `DynamicalSystem`, you need to consult the main tutorial of the
+[DynamicalSystems.jl library](https://juliadynamics.github.io/DynamicalSystemsDocs.jl/dynamicalsystems/stable/tutorial/).
+For this tutorial we will use a modified Lorenz-like system with equations
+
+```math
+\dot{x} & = y - x \\
+\dot{y} & = -x*z + b*|z| \\
+\dot{x} & = x*y - a \\
+```
+
+which we define in code as
+
+````@example tutorial
+using Attractors
+using OrdinaryDiffEq # for accessing advanced ODE Solvers
+
+function modified_lorenz_rule(u, p, t)
+    x, y, z = u; a, b = p
+    dx = y - x
+    dy = - x*z + b*abs(z)
+    dz = x*y - a
+    return SVector(dx, dy, dz)
+end
+
+p0 = [5.0, 0.1] # parameters
+u0 = [-4.0, 5, 0] # state
+ds = CoupledODEs(modified_lorenz_rule, u0, p0;
+    diffeq = (alg = Vern9(), abstol = 1e-9, reltol = 1e-9, dt = 0.01)
+)
+````
+
+## Finding attractors
+
+There are two major methods for finding attractors in dynamical systems.
+Explanation of how they work is in their respective docs.
+
+1. [`AttractorsViaRecurrences`](@ref).
+2. [`AttractorsViaFeaturizing`](@ref).
+
+You can consult [Datseris2023](@cite) for a comparison between the two.
+
+As far as the user is concerned, both algorithms are part of the same interface,
+and can be used in the same way. The interface is extendable as well,
+and works as follows.
+
+First, we create an instance of such an "attractor finding algorithm",
+which we call `AttractorMapper`. For example, [`AttractorsViaRecurrences`](@ref)
+requires a tesselated grid of the state space to search for attractors in.
+It also allows the user to tune some meta parameters, but in our example
+they are already tuned for the dynamical system at hand. So we initialize
+
+````@example tutorial
+grid = (
+    range(-10.0, 10.0; length = 100), # x
+    range(-15.0, 15.0; length = 100), # y
+    range(-15.0, 15.0; length = 100), # z
+)
+
+mapper = AttractorsViaRecurrences(ds, grid;
+    consecutive_recurrences = 1000, attractor_locate_steps = 1000,
+    consecutive_lost_steps = 100,
+)
+````
+
+Then, this `mapper` can be given to several functions for finding attractors
```

Comment:
Here, I would start with a simple use of the mapper and explain that this is a function from the state space to a label: 

`mapper(u0) -> identifier`

And each newly found attractor gets a new label.